### DISCUSSION TITLE
mutator is not working while updating the data

### DISCUSSION THREAD

**Discussion by Shankar-Tom** [laravel/framework#45244] [Q&A]
protected function password():Attribute
    {
        return Attribute::make(
            set: fn ($value) => bcrypt($value),
        );
    }

I am using this mutator funtion to encrypt my password while storing to database it encrypt the password when i create a new record but not encrypting my password while i am updating my password using update function

**Comment by henzeb** [comment]
have you tried adding a get?

**Reply by Shankar-Tom** [reply]
Yeah i tried but its encrypting the password while storing on data it only encrypting while returning 
and if i stored the encrypted password ,auth attempt is returning false
